src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -52,6 +53,8 @@ public function __construct(array $mockedNamespaces = array())
             Blacklist::$blacklistedClassNames['\Symfony\Bridge\PhpUnit\Legacy\SymfonyTestsListenerTrait'] = 2;
         }
 
+        $enableDebugClassLoader = \class_exists('Symfony\Component\Debug\DebugClassLoader');
```

Comment:
(as a reminder, because want to use it when testing branch 2.8, the bridge is still compatible with PHP 5.3, so we cannot use `::class` here)